portfolio/src/main/webapp/index.html

```diff
@@ -6,31 +6,16 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
 
-    <style>
-        .name {
-            border-radius: 15px;
-            line-height: 60px;
-            color : #000066;
-        }
+  </head>
+  
+  <body class = "background">
 
-        .name-1 {
-            top: 71%;
-            left: 90%;
-        }
-    </style>
+    <button id = "contactButton" class="button"><b>Contact</b></button>
```

Comment:
If you want you should be able to set the styling of all buttons and so not need to set class button

Reply:
I am thinking that in the future I can have some other buttons that have different specs so I can have 2 seperate classes. I am not sure if this is the right way tho

Reply:
Also, the current buttons have some attributes in common and I thought having them inherit a class would be a better way of organizing the code, since I won't have to write for each button all the attributes (ultimately, having less code).

Reply:
css allows to use various types of selectors to apply style e.g: class, id, element type... That also can be combined.

button { ... } - Will apply style to all buttons on the page.

.button {...}- apply style to all element with this style name.